app/models/legal_aid_application.rb

```diff
@@ -138,6 +140,26 @@ def submission_date
     transaction_period_finish_at.to_date
   end
 
+  # This is here so that this method can be stubbed in the ccms payload integration tests
+  # it will be later replaced by an opponents association to load a dummy opponent for CCMS
+  # injection
```

Comment:
In some classes there are these long explanations about why static content is being returned. In others (for example the Opponent model) there are similar methods with no comments. Can we mark them all with a simple easily identifiable comment. I'd suggest using a TODO that many IDE will find easily:
```ruby
def opponnents
  nil  # TODO: Replace with CCMS data
end
```
or
```ruby
def opponnents
  nil  # TODO: CCMS placeholder
end
```
And use it everywhere it's relevant.